### PR TITLE
Add deepARG

### PR DESCRIPTION
**Pull request by jfy133**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/deeparg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - deeparg
+  commands:
+    - deeparg -h
+
+about:
+  home: https://bitbucket.org/gusphdproj/deeparg-ss/
+  license: MIT
+  license_family: MIT
+  license_file: "LICENSE.txt"
+  summary: A deep learning based approach to predict Antibiotic Resistance Genes (ARGs) from metagenomes.
```
Comment (jfy133) [inline recipes/deeparg/meta.yaml:46]:
```suggestion
  summary: A deep learning based approach to predict Antibiotic Resistance Genes (ARGs) from metagenomes
```

**Review by jfy133 — COMMENTED** [review]
To try and restart ci

**Comment by jfy133** [comment]
@bioconda/core the circleci workflow are timing out during container preparation (even though it seems the tests pass). It appears the 'longest' step is resolving the building environment... do you have any idea why this could be? The dependcies for this seem to be pretty straight forward... (if I'm understanding the issue correctly)

**Comment by dpryan79** [comment]
I do not pretend to understand what conda was thinking while solving this and why it made such a simple resolution so complicated.

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**